src/FindTranslocations.cpp

```diff
@@ -15,9 +16,127 @@
 
 #include "data_structures/Translocation.h"
 #include  <boost/program_options.hpp>
+#include "api/BamWriter.h"
+
 namespace po = boost::program_options;
 
 
+//returns a bam file contaning the reads located in a given region of the genome
+void TranslocationBAM(string BamFileName,string outputFileHeader,string positionData){
+	int chr[2];
+	int startPos[2];int endPos[2];
+	int arguments;
+
+	//processing the command line input
+	string buf; 
+	stringstream ss(positionData);
+	vector<string> tokens;
+	while (ss >> buf){tokens.push_back(buf);}
+	arguments=tokens.size();
+
+
+	string tmpstr;
+	if(arguments == 6){
```

Comment:
This is in general is not recommended, if you have many of these checks in a function makes the code difficult to maintain. Imagine that in three months I want to add a new variable, at this point I need to increment 6 to 7